meerk40t/camera/gui/camerapanel.py

```diff
@@ -1,5 +1,6 @@
 import platform
 
+import cv2
```

Comment:
This is actually somewhat problematic. OpenCV is/was not a requirement however, since services exists now this doesn't appear to be a problem anymore. There was a time historically that the gui for the camera was distinct from the background backend bit of the camera as it was intended to (though a bit wrongly) act like a plugin module. If OpenCV is not installed the Camera does not exist and everything just sort of acts normally.

I'll have to check.

Worked fine. Disabled the camera and lost the camera app exactly as it should. Though I do notice that the warning it throws is wrong.
```
OpenCV is not installed. Disabling Camera. Install with:
	pip install opencv-python headless
```
There's actually a hyphen between python and headless there rather than a space. But, that's not on you.

Reply:
I've encapsulated cv2 a bit (even if it's not necessary) and the associated UBS port for cam 0 to 4 are saved beyond a session - so it should be okay in principle to detect cameras only once.
To reset the association to the default just call "Identify cameras" again

Reply:
I checked it was fine.